setup.cfg

```diff
@@ -0,0 +1,12 @@
+[wheel]
+universal = 1
+
+[flake8]
+ignore =
+max-line-length = 120
+max-complexity = 10
```

Comment:
I think it's too much. What is the default value?

Reply:
there was the same value for `aibolit`.
default is 79